playbooks/openstack-osh_hostconfig.yml

```diff
@@ -8,6 +8,25 @@
     - name: Ensure all the openstack mechanism vars are loaded
       include_vars: "{{ playbook_dir }}/../vars/deploy-on-openstack.yml"
 
+    - name: Collect repo files for removal
+      find:
+        paths: "/etc/zypp/repos.d/"
+        patterns: "*.repo"
+      register: repo_files
+
+    - name: Clear existing repo config
+      file:
+        path: "{{ item['path'] }}"
+        state: absent
+      with_items: "{{ repo_files['files'] }}"
```

Comment:
Always interesting to see how many lines of ansible you need to do simple tasks. wow